action.yml

```diff
@@ -31,7 +31,7 @@ runs:
   image: 'docker://ghcr.io/loveholidays/gitops-action-yaml-updater:v1.7.0'
   args:
     - ${{ inputs.mode }}
-    - ${{ inputs.container-name }}
+    - ${{ inputs.container-names }}
```

Comment:
This will need to stay as container-name to avoid having to change it everywhere

Reply:
and keep the value comma separated names? just keeping the variable name?

Reply:
Yeah - you can go with the plural but you'll need to update it anywhere it's used